### PR TITLE
remove wiki bzr cvs darcs fossil svn support

### PR DESCRIPTION
**Pull request by steckerhalter**
upstream `package-build` has removed the wiki fetcher hence we do the same

### CODE REVIEW

**Comment by steckerhalter** [comment]
@wasamasa @vyp does anyone have time to review/merge this PR? thanks

File: quelpa.el [hunk 1]
```diff
@@ -483,8 +486,7 @@ If there is an error and no existing checkout return nil."
            'package-build
            `(:url ,quelpa-melpa-repo-url :files ("*"))
            quelpa-melpa-dir)
-        (error (quelpa-message t "failed to checkout melpa git repo: `%s'" (error-message-string err))
-               (file-exists-p (expand-file-name ".git" quelpa-melpa-dir))))))
```
Comment (wasamasa) [inline quelpa.el]: Why did you remove the argument line?

**Comment by wasamasa** [comment]
Looks good to me and works as expected.

File: quelpa.el (hunk 1 above)
Comment (steckerhalter) [reply quelpa.el]: it was an ARG for `error`, but there is no format string for `error` so it was not used anyway if I'm not mistaken...

Comment (wasamasa) [reply quelpa.el]: Yeah, I'm mostly wondering what the intention was here, it seems like you intended to print both the original error message and the path to `quelpa-melpa-dir`.

Comment (steckerhalter) [reply quelpa.el]: I thought so too. I figured to display the path as well doesn't make much sense.
